admin-perspective/4.4/notification-drawer/notification-drawer.md

```diff
@@ -0,0 +1,28 @@
+# Notification Drawer
+
+## Masthead Icon
+![](img/notifications-closed.png)
+- When alerts are firing, a special signifier icon appears in the masthead. Clicking on the icon opens the notification drawer.
+
+## Critical Alerts
+![](img/notifications-open.png)
+- As this drawer is an endopoint for alertmanager, silenced and pending alerts do not appear here.
+- Clicking on an alert takes the user to the details page for the alert.
+- The alertmanager config could potentially be set up to send non-critical alerts here, but by default that would not be the case.
+- The critical alerts group should be expanded by default  and all others should be collapsed.
+
+## Available Updates
+![](img/notifications-updates.png)
+- Updates for the cluster or for specific operators could also be listed here.
+- Clicking on one of these cards would take the user to a page with information about the updates.
+- Action buttons to install the update immediately could also be provided.
+
+## Empty State
+![](img/notifications-empty.png)
+- When there are no critical alerts firing, the `Critical alerts` group should still be present but should contain an empty state pattern.
+- The `Available updates` category and any additional categories should be hidden when they are empty.
```

Comment:
Should we show progress during an upgrade?

Reply:
If we have that capability, I think that would be great!

Reply:
I am concerned that this tray should be strongly focused on things that justify an alert.  An upgrade is important, but only when it starts failing.  We could come up with lots of "current status" things to show here, but I think we should start by only focusing on alerts until the alert system is a) properly integrated, b) well used, and c) covers the use cases.

Reply:
I believe we currently put an icon in the masthead to indicate that there's an available update. Combining that messaging into this space would keep upgrades from competing for people's attention because a pressing alert would be surfaced above the update notification. It also reduces the amount of stuff in the masthead.